src/bobbit/modules/tell.py

```diff
@@ -0,0 +1,34 @@
+# tell.py
+
+# Metadata
+NAME    = 'tell'
+PATTERN = r''
+ENABLE  = True
+USAGE   = '''Usage: !tell <user> <message>
+This queues a message to send to a user the next time they are active (ie.
+the next time they send a message).
+'''
+
+MAILBOX = {}
+
+# Command
+
+async def tell(bot, message):
+    if message.body.startswith("!tell"):
+        tell_message = message.body[5:].split()
+        if tell_message[0] in MAILBOX:
```

Comment:
If we make `MAILBOX` a `defaultdict`, we can just do an append, no need for the `if`